src/server/api/controllers/retrieve-first-question.controller.js

```diff
@@ -0,0 +1,22 @@
+const knex = require('../../config/db');
+const Error = require('../lib/utils/http-error');
+
+const getFirstQuestionById = async (id) => {
+  try {
+    const modules = await knex('questions')
+      .innerJoin('game_factory', 'questions.fk_game_id', '=', 'game_factory.id')
```

Comment:
Hi @SwathiSankararaman. Consider that the name of the foreign key in questions was changed to fk_game_factory_id in the PR https://github.com/HackYourFuture-CPH/fp-class12/pull/158 :) (already merged to develop)

Reply:
Yes I changed that 👍